templates/CRM/Case/Page/CustomDataView.tpl

```diff
@@ -16,36 +16,31 @@
       {$cd_edit.title}
     </div>
     <div class="crm-accordion-body">
-      {foreach from=$cd_edit.fields item=element key=field_id}
+      {if !empty($cd_edit.fields)}
```

Comment:
is there a reason for needing this if here?

Reply:
Yes, because the `<table>` tag would only be needed if there are data for rows in it and would cause an empty `<table></table>` element otherwise.